packages/asset-service/src/service/descriptions.json

```diff
@@ -1,6 +1,7 @@
-  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains MIM and 3CRV. 3CRV is another curve LP token that consists of DAI, USDT, and USDC. To acquire crvMIM, deposit MIM, DAI, USDT, and/or USDC on [Curve](https://curve.fi/mim/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg."
+  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains MIM and 3CRV. 3CRV is another curve LP token that consists of DAI, USDT, and USDC. To acquire crvMIM, deposit MIM, DAI, USDT, and/or USDC on [Curve](https://curve.fi/mim/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg.",
+  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on [ShapeShift](https://app.shapeshift.com/defi/earn) to earn yield. This liquidity pool contains [MIM](https://app.shapeshift.com/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) and [UST](https://app.shapeshift.com/accounts/eip155:1:0xa3f620b1b4c1e3a14880fb434b3a1a8792b7ab75). TerraUSD ([UST](https://app.shapeshift.com/assets/eip155:1/erc20:0xa693b19d2931d498c5b318df961919bb4aee87a5)) is the decentralized and algorithmic stable coin of the Terra blockchain. To acquire Curve MIM-UST, deposit [MIM](https://app.shapeshift.com/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) and [UST](https://app.shapeshift.com/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) on [Curve](https://curve.fi/factory/48/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg."

```

Comment:
```suggestion
  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on [ShapeShift](/defi/earn) to earn yield. This liquidity pool contains [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) and [UST](/accounts/eip155:1:0xa3f620b1b4c1e3a14880fb434b3a1a8792b7ab75). TerraUSD ([UST](/assets/eip155:1/erc20:0xa693b19d2931d498c5b318df961919bb4aee87a5)) is the decentralized and algorithmic stable coin of the Terra blockchain. To acquire Curve MIM-UST, deposit [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) and [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) on [Curve](https://curve.fi/factory/48/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg."
```

Reply:
@0xTomas - pretty much everything is wrong.

* all the internal links must only include the pathname https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname
* your first UST link shows your address and goes to your eth account
* your second UST goes to the wormhole contract, not the asset https://docs.wormholenetwork.com/wormhole/deposit-and-withdraw-wormhole-wrapped-assets-from-ftx/tutorial-how-to-withdraw-deposit-ust-wormhole-from-ftx
* your last UST link is the correct wrapped UST asset

please make the above changes and resubmit